frontend/app/src/components/StatusWidget/StatusWidget.tsx

```diff
@@ -361,7 +355,8 @@ class StatusWidget extends PureComponent<StatusWidgetProps, State> {
 
     // check if current date between 12/31 and 1/06 to render correct gif
     const isNewYears = StatusWidget.isNewYears()
-    const runningSrc = isNewYears ? newYearsRunning : iconRunning
+    // Use paths in the public folder to keep the paths relative
+    const runningSrc = isNewYears ? "fireworks.gif" : "icon_running.gif"
```

Comment:
Balloon & snow still have their assets in the `lib/src/img/ballons` / `lib/src/img/snow` folders and load it via `import Balloon0 from "@streamlit/lib/src/assets/img/balloons/balloon-0.png"`. Whats the reason why this works fine for balloon / snow in lib, but not in for these `gif` images in app?

Reply:
So when working through this, I was working in dev mode, which, in tests creates image urls at the root `/src/assets/img/icon_running.gif`. This broke tests mainly because we are testing whether it's the runnning man or the fireworks. We can easily update the tests.

I now have investigated release mode because we want the url to be relative. It looks like it provides the full url with a base path and everything. See image.
<img width="750" alt="image" src="https://github.com/user-attachments/assets/cda8691d-7d68-4e5c-b326-f125c63c3709">

I'm going to deploy a package to community cloud to see how they handle it. I imagine it will not be an issue because I don't believe baseURLPath will work on Community Cloud. But if this all works out, the solution may be to just undo this cause it should work fine.

Reply:
I tried it with the proxy example as well: https://github.com/lukasMasuch/st-proxy-app and it seems to work fine out of the box :) I believe the URL prefix is added dynamically at runtime.